version.php

```diff
@@ -29,10 +29,10 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2020110301;  // Plugin version.
-$plugin->requires = 2017111300; // require Moodle version (3.4).
+$plugin->version = 2024041000;  // Plugin version.
```

Comment:
I think leave this as a micro bump, we cannot ever let this overtake what is in the upstream repo which is currently 2020110300

Reply:
Thanks @brendanheywood I just checked the upstream repo and their version is `2020110300` but in our fork the version is `2020110301` so I've rebased and updated the version number to `2020110301.01`